tests/e2e/pact/specs/sandbox/prepare.successes.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { InteractionObject, Matchers } from "@pact-foundation/pact"
+import * as jestpact from "jest-pact"
+import supertest from "supertest"
+import * as TestResources from "../../resources/test-resources"
+import * as LosslessJson from "lossless-json"
+import * as uuid from "uuid"
+import { basePath, getStringParameterByName, pactOptions } from "../../resources/common"
+import {fhir} from "@models"
+
+jestpact.pactWith(
+  pactOptions("sandbox", "prepare"),
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
+  async (provider: any) => {
+    const client = () => {
+      const url = `${provider.mockService.baseUrl}`
+      return supertest(url)
+    }
+
+    describe("prepare sandbox e2e tests", () => {
+      test.each(TestResources.prepareCaseGroups)("should be able to prepare a %s message", async (desc: string, request: fhir.Bundle, response: fhir.Parameters) => {
+        const apiPath = `${basePath}/$prepare`
+        const requestStr = LosslessJson.stringify(request)
+        const requestId = uuid.v4()
+        const correlationId = uuid.v4()
+
+        const interaction: InteractionObject = {
+          state: "is not authenticated",
+          uponReceiving: `a request to prepare a ${desc} message`,
+          withRequest: {
+            headers: {
+              "Content-Type": "application/fhir+json; fhirVersion=4.0",
+              "X-Request-ID": requestId,
+              "X-Correlation-ID": correlationId
+            },
+            method: "POST",
+            path: apiPath,
+            body: JSON.parse(requestStr)
+          },
+          willRespondWith: {
+            headers: {
+              "Content-Type": "application/json",
+              "X-Request-ID": requestId,
+              "X-Correlation-ID": correlationId
+            },
+            body: {
```

Comment:
thoughts about loading the sandbox return files in the body check?

Reply:
We cant use exact response files as the digest and timestamp are different every time you call the api due to the timestamp being added per api call and this forming part of the digest as well as returned in a field.

We are using the sandbox responses to generate examples of what these fields look like and the below code ensures that the body will be identical except for digest and timestamp values which are checked that they contain any string value

Reply:
sorry yea just realised ive been an absolute clown, the sandbox responses are in v3 anyway